fang/README.md

```diff
@@ -392,44 +396,49 @@ Set sleep params with worker pools `TypeBuilder` in both modules.
 5. Create a new Pull Request
 
 ### Running tests locally
+
 - Install diesel_cli.
-```
+
+```sh
 cargo install diesel_cli --no-default-features --features "postgres sqlite mysql"
 ```
+
 - Install docker on your machine.
 
-- Run a Postgres docker container. (See in Makefile.)
-```
-make db_postgres
-```
+- Install SQLite 3 on your machine.
 
-- Run the migrations
-```
-make diesel_postgres
-```
+- Setup databases for testing.
 
-- Run tests
-```
-make tests
+```sh
+make -j db
 ```
 
-- Run dirty//long tests, DB must be recreated afterwards.
-```
-make ignored
+- Run tests. `make db` does not need to be run in between each test cycle.
+
+```sh
+make -j tests
 ```
 
-- Kill the docker container
+- Run dirty/long tests.
+
+```sh
+make -j ignored
 ```
-make stop
+
+- Take down databases.
+
+```sh
+make -j stop
 ```
 
+The `-j` flag in the above examples is not necessary but highly recommended.
```

Comment:
I think `-j` specified the number of threads, correct?

why do we need this parameter?

Reply:
I think , it is because it is processed in multiple threads which is cool.

Reply:
`-j` can be used to specify number of threads, but when used alone it just tells `make` to use as many threads as needed. Without it, `make` does not use parallelism.

Reply:
would be nice to mention this in the readme

Reply:
Added a small clarification to the `README.md`.